osu.Game/Screens/SelectV2/BeatmapLeaderboardScore.cs

```diff
@@ -41,9 +41,9 @@
 using osuTK.Graphics;
 using CommonStrings = osu.Game.Localisation.CommonStrings;
 
-namespace osu.Game.Screens.SelectV2.Leaderboards
+namespace osu.Game.Screens.SelectV2
 {
-    public partial class LeaderboardScoreV2 : OsuClickableContainer, IHasContextMenu, IHasCustomTooltip<ScoreInfo>
+    public partial class BeatmapLeaderboardScore : OsuClickableContainer, IHasContextMenu, IHasCustomTooltip<ScoreInfo>
```

Comment:
Don't really get this one, this was `LeaderboardScoreV2` to distinguish against old-song-select `LeaderboardScore`, so kinda not sure what's with the `Beatmap` qualifier here. Also note that this is used in daily challenge.

Not hugely fussed though, probably can stay. I'd hope the old `LeaderboardScore` can be old yeller'd at some point because it's pretty horrific with the way it's been contorted to fit playlists.

Reply:
Not sure why `Beatmap` was added here, I'm fine to drop that if it makes things better.

Reply:
Let's just go with it as is.

Reply:
I guess it was kind of there to group this in follow-up work:

![JetBrains Rider-EAP 2025-04-16 at 09 54 39](https://github.com/user-attachments/assets/3f6029b3-78f0-4df5-9329-49a05a698946)